---
author: kraman
category:
  - fba
  - tuberculosis
cover:
  alt: MycolicAcidPathway
  image: /kraman/lab/wp-content/uploads/2013/05/MycolicAcidPathway.png
date: "2005-10-14T04:30:24+00:00"
tag:
  - fba
  - tuberculosis
title: Flux balance analysis of Mycolic Acid Pathway

---
_[![MycolicAcidPathway](/kraman/lab/wp-content/uploads/2013/05/MycolicAcidPathway-212x300.png)](/kraman/lab/wp-content/uploads/2013/05/MycolicAcidPathway.png) M. tuberculosis,_ a deadly human pathogen, owes many of its unique qualities to its thick, waxy coat, containing fatty acids called mycolic acids. Several front-line drugs used for treating tuberculosis indeed inhibit mycolic acid synthesis. Understanding the biochemical pathway that makes these compounds is therefore of great interest. Availability of the genome sequence and various computational methods enable us to study pathways as whole functional units, rather than having to infer from the study of individual proteins.

Here, we present a comprehensive identification of the components of the mycolic acid pathway and represent it mathematically based on reaction stoichiometry. Such models are amenable to perturbations and simulations using flux balance analysis, allowing the study of pathways from a metabolic capacity perspective, and yielding information about reaction fluxes. The perturbations studied here are in silico gene knock-outs and drug effects, which led us to identify genes essential to the pathway and hence for survival of the pathogen. The results are in good agreement with essentiality determined through experimental genetics. Such essential genes can be good targets for drug design, especially when they do not have homologues in the human proteome. FBA followed by sequence analyses have resulted in identification of potential anti-tubercular drug targets.

We present a comprehensive model of mycolic acid synthesis in the pathogen _M. tuberculosis_ involving 197 metabolites participating in 219 reactions catalysed by 28 proteins. Flux balance analysis (FBA) has been performed on the MAP model, which has provided insights into the metabolic capabilities of the pathway. In silico systematic gene deletions and inhibition of InhA by isoniazid, studied here, provide clues about proteins essential for the pathway and hence lead to a rational identification of possible drug targets. Feasibility studies using sequence analysis of the _M. tuberculosis_ H37Rv and human proteomes indicate that, apart from the known InhA, potential targets for anti-tubercular drug design are AccD3, Fas, FabH, Pks13, DesA1/2, and DesA3. Proteins identified as essential by FBA correlate well with those previously identified experimentally through transposon site hybridisation mutagenesis. This study demonstrates the application of FBA for rational identification of potential anti-tubercular drug targets, which can indeed be a general strategy in drug design. The targets, chosen based on the critical points in the pathway, form a ready shortlist for experimental testing.

_**Original Paper**_: \[bibtex file=karthikraman-publications.bib key=Raman2005Flux\]
