---
title: 'iCOMIC: a graphical interface-driven bioinformatics pipeline for analyzing
  cancer omics data'
date: '2022-09-01'
draft: true
publishDate: '2024-08-30T15:53:27.606927Z'
authors:
- Anjana Anilkumar Sithara
- Devi Priyanka Maripuri
- Keerthika Moorthy
- Sai Sruthi Amirtha Ganesh
- Philge Philip
- Shayantan Banerjee
- Malvika Sudhakar
- Karthik Raman
publication_types:
- '2'
abstract: Despite the tremendous increase in omics data generated by modern sequencing
  technologies, their analysis can be tricky and often requires substantial expertise
  in bioinformatics. To address this concern, we have developed a user-friendly pipeline
  to analyze (cancer) genomic data that takes in raw sequencing data (FASTQ format)
  as input and outputs insightful statistics. Our iCOMIC toolkit pipeline featuring
  many independent workflows is embedded in the popular Snakemake workflow management
  system. It can analyze whole-genome and transcriptome data and is characterized
  by a user-friendly GUI that offers several advantages, including minimal execution
  steps and eliminating the need for complex command-line arguments. Notably, we have
  integrated algorithms developed in-house to predict pathogenicity among cancer-causing
  mutations and differentiate between tumor suppressor genes and oncogenes from somatic
  mutation data. We benchmarked our tool against Genome In A Bottle benchmark dataset
  (NA12878) and got the highest F1 score of 0.971 and 0.988 for indels and SNPs, respectively,
  using the BWA MEM—GATK HC DNA-Seq pipeline. Similarly, we achieved a correlation
  coefficient of r = 0.85 using the HISAT2-StringTie-ballgown and STAR-StringTie-ballgown
  RNA-Seq pipelines on the human monocyte dataset (SRP082682). Overall, our tool enables
  easy analyses of omics datasets, significantly ameliorating complex data analysis
  pipelines.
featured: false
publication: '*NAR Genomics and Bioinformatics*'
doi: 10.1093/nargab/lqac053
links:
- name: URL
  url: https://doi.org/10.1093/nargab/lqac053
---

